build/script/steamcmd

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env bash
 
-if [ `whoami` != "$STEAM_USER" ]; then
-	exec gosu $STEAM_USER $0 "$@"
-fi
-
```

Comment:
I mean, if someone runs `docker exec -u root my-container steamcmd` on purpose, it should use the root user and not use gosu, or? Or am I missing a special use-case?